### PR TITLE
add Chapel 1.13 release branch dates to perf utilities

### PR DESCRIPTION
**Pull request by awallace-cray**

### CODE REVIEW

**Comment by awallace-cray** [comment]
@bradcray please review or wait until the time is right. Thanks.

**Comment by bradcray** [comment]
Looks OK to me!

File: util/test/perf/perfgraph.js [hunk 1]
```diff
@@ -71,6 +71,10 @@ var branchInfo = [
                     "releaseDate": "2015-10-01",
                     "branchDate" : "2015-09-24",
                     "revision" : -1}
```
Comment (ronawho) [inline util/test/perf/perfgraph.js]: Need to add a comma. 

`"revision" : -1}` => ` "revision" : -1},`

Comment (bradcray) [inline util/test/perf/perfgraph.js]: Geez, and I even thought to look for missing commas...

I'm just trying to train people not to ask me for reviews... :P

**Comment by awallace-cray** [comment]
Comma is added. Sorry about that.

**Pull request merged**